packages/muelu/research/mmayr/composite_to_regions/src/SetupRegionSmoothers_def.hpp

```diff
@@ -536,6 +518,7 @@ void smootherSetup(RCP<Teuchos::ParameterList> params,
   }
   case 3: // Chebyshev
   {
+    relaxationSmootherSetup(params, revisedRowMapPerGrp, regionGrpMats, rowImportPerGrp);
```

Comment:
I guess, the method `relaxationSmootherSetup()` could/should be renamed to something like `smootherSetup()` or `precomputeInverseDiagonal()`, since this is all that it does. In particular, it is not specific to relaxation smoothers anymore.

Furthermore, the name of the parameter to store the inverse diagonal in the parameter list could/should be renamed as well, since it is not limited to the case of a "relaxation smoother".

Reply:
Let's rename the parameter list entry to `smoother: inverse diagonal`.

Reply:
Done.